web/docs/hosting/fly-io.md

```diff
@@ -56,23 +56,24 @@
         :
 
         [[services]]
-          internal_port = 8081
+          internal_port = "8080"
         :
         :
         ```
 
 1. Set the API secret and APP_KEY environment variables for your app:
 
     ```shell
-    flyctl secrets set SHOPIFY_API_SECRET=ReplaceWithSECRETFromPartnerDashboard
-    php artisan key:generate --show
+    cd .. && yarn print-env
+    flyctl secrets set SHOPIFY_API_SECRET=ReplaceWithSECRETFromEnvCommand
+    cd web && php artisan key:generate --show
```

Comment:
```suggestion
    (cd .. && yarn print-env)
    flyctl secrets set SHOPIFY_API_SECRET=ReplaceWithSECRETFromEnvCommand
    php artisan key:generate --show
```
It's a bit of a nit ... and maybe if people forget the `(`/`)` they'll get confused ... 🤔

Reply:
🔥